File: example/bot.py
import sys

board_size, turns = map(int, input().split())

players = int(input())

for _ in range(players):
    row, col = map(int, input().split())

for _ in range(turns):
    agents = int(input())

    for _ in range(agents):
        row, col, object, dist, rot = input().split()

    for _ in range(agents):
        print("FIRE", flush=True)

File: example/bot.cpp
#include <iostream>
using namespace std;

int main() {
  int board_size, turns;
  cin >> board_size >> turns;

  for (int t = 0; t < turns; t++) {
    int agents;
    cin >> agents;

    for (int i = 0; i < agents; i++) {
      int row, col;
      string type;
      int dist;
      char rot;
      cin >> row >> col >> type >> dist >> rot;
    }

    for (int i = 0; i < agents; i++) {
      cout << "FIRE\n";
    }
    cout.flush();
  }
}
